scripts/migration/yq.sh

```diff
@@ -38,6 +38,19 @@ yq_move() {
   fi
 }
 
+yq_xmove() {
```

Comment:
I think I would rather like
```suggestion
yq_move_file() {
```

To make it a bit more descriptive

Reply:
`yq_move_to_file` as to not imply that it moves files themselves?

Reply:
Yea, that sounds good